sql/updates/world/3.3.5/2020_xx_xx_xx_world.sql

```diff
@@ -0,0 +1,73 @@
+-- Deathbound Ward Guid
+SET @DW_GUID_FIRST := 200987;
+SET @DW_GUID_SECOND := 201043;
+SET @DW_GUID_THIRD := 201108;
+SET @DW_GUID_FOURTH := 201130;
+
+-- Deathbound Ward Waypoints
+SET @DW_WAYPOINT_FIRST := 2009870;
+SET @DW_WAYPOINT_SECOND := 2010430;
+SET @DW_WAYPOINT_THIRD := 2011080;
+SET @DW_WAYPOINT_FOURTH := 2011300;
+
+DELETE FROM `smart_scripts` WHERE `source_type` = 0 AND `entryorguid` IN(-@DW_GUID_FIRST, -@DW_GUID_SECOND, -@DW_GUID_THIRD, -@DW_GUID_FOURTH);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-@DW_GUID_FIRST, 0, 0, 0, 0, 0, 100, 512, 3000, 6000, 6000, 8000, 0, 11, 71021, 2, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Deathbound Ward - In Combat - Cast \'Saber Lash\''),
+(-@DW_GUID_FIRST, 0, 1, 0, 0, 0, 100, 0, 6000, 8000, 6000, 12000, 0, 11, 71022, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Deathbound Ward - In Combat - Cast \'Disrupting Shout\''),
+(-@DW_GUID_FIRST, 0, 2, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 53, 0, @DW_WAYPOINT_FIRST, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Deathbound Ward - On Data Set 1 1 - Start Waypoint'),
+(-@DW_GUID_FIRST, 0, 3, 0, 63, 0, 100, 0, 0, 0, 0, 0, 0, 55, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Deathbound Ward - On Just Created - Stop Waypoint'),
```

Comment:
Why?

Reply:
Now that you mention it, I admit that I also don't know why it should be necessary to stop any activities 'On Just Created'.

Reply:
@offl why i dont stop wayponts?

Reply:
You don't need to stop waypoints On Just Created because On Just Created they don't have waypoints
Waypoints are loaded only when traps are activated

Reply:
> You don't need to stop waypoints On Just Created because On Just Created they don't have waypoints
> Waypoints are loaded only when traps are activated

It will not finish moving after the end of waypoints?

Reply:
Nope

Reply:
@offl And I can safely delete it?

Reply:
Sure

Reply:
Done